Typescript Error

**Error**:
```
error TS2605: JSX element type 'StripeCheckout' is not a constructor function for JSX elements.
  Types of property 'setState' are incompatible.
    Type '{ <K extends never>(f: (prevState: void, props: StripeCheckoutProps) => Pick<void, K>, callback?:...' is not assignable to type '{ <K extends never>(f: (prevState: {}, props: any) => Pick<{}, K>, callback?: () => any): void; <...'.
      Types of parameters 'f' and 'f' are incompatible.
        Type '(prevState: {}, props: any) => Pick<{}, any>' is not assignable to type '(prevState: void, props: StripeCheckoutProps) => Pick<void, any>'.
          Types of parameters 'prevState' and 'prevState' are incompatible.
            Type 'void' is not assignable to type '{}'.
```

**Use**:
```
<StripeCheckout
    name="name"
    description="description"
    image="/picture.jpg"
    ComponentClass="div"
    stripeKey="pk_test_..."
    token={token => console.log("Token received!")}
>
    <RaisedButton label="Pay for Somethin'" primary />
</StripeCheckout>
```

**Solution**:
In index.d.ts, change
```
export default class StripeCheckout extends React.Component<StripeCheckoutProps, void> {}
```
to
```
export default class StripeCheckout extends React.Component<StripeCheckoutProps, {}> {}
```
or
```
export default class StripeCheckout extends React.Component<StripeCheckoutProps, Object> {}
```

I've fixed the same problem in [material-ui-number-input](https://github.com/NoHomey/material-ui-number-input) and will be submitting a PR momentarily.

--- a/index.d.ts
+++ b/index.d.ts
@@ -60,5 +60,5 @@ declare module "react-stripe-checkout" {
     }
 
     import React = require("react");
-    export default class StripeCheckout extends React.Component<StripeCheckoutProps, void> {}
+    export default class StripeCheckout extends React.Component<StripeCheckoutProps, Object> {}
 }

